canonical/ubuntu-pro-client
# Check onlySeries on reboot

## Why is this needed?

Runs onlySeries check on machine reboot instead of a timer service



Fixes: #3189 

## Test Steps

Run the tox integration test for the `features/ubuntu_upgrade` file to check if the onlySeries check runs successfully on reboot




---
